monai/transforms/post/dictionary.py

```diff
@@ -56,4 +56,35 @@ def __call__(self, data):
         return d
 
 
+class KeepLargestConnectedComponentd(MapTransform):
+    """
+    dictionary-based wrapper of :py:class:monai.transforms.utility.array.KeepLargestConnectedComponent.
+    """
+
+    def __init__(self, keys, applied_values, independent=True, background=0, connectivity=None):
+        """
+        Args:
+            keys (hashable items): keys of the corresponding items to be transformed.
+                See also: :py:class:`monai.transforms.compose.MapTransform`
+            applied_values (list or tuple of int): number list for applying the connected component on.
+                The pixel whose value is not in this list will remain unchanged.
+            independent (bool): consider several labels as a whole or independent, default is `True`.
+                Example use case would be segment label 1 is liver and label 2 is liver tumor, in that case
+                you want this "independent" to be specified as False.
+            background (int): Background pixel value. The over-segmented pixels will be set as this value.
+            connectivity (int): Maximum number of orthogonal hops to consider a pixel/voxel as a neighbor.
+                Accepted values are ranging from  1 to input.ndim. If ``None``, a full
+                connectivity of ``input.ndim`` is used.
+        """
+        super().__init__(keys)
+        self.converter = KeepLargestConnectedComponent(applied_values, independent, background, connectivity)
+
+    def __call__(self, data):
+        d = dict(data)
+        for idx, key in enumerate(self.keys):
+            d[key] = self.converter(d[key])
```

Comment:
BTW, in order to keep the original model output for other post transforms or some additional computation, let's add a new key to store the data in `key_{postfix}`.
Refer to: https://github.com/Nic-Ma/MONAI/blob/417-add-ConvertForMetrics-post-transform/monai/transforms/post/dictionary.py#L112

Thanks.

Reply:
done